src/index.html

```diff
@@ -24,7 +24,8 @@ <h1 class="h0 mt0 mb0 white inline">AngularJS PDF Viewer</h1>
         delegate-handle="my-pdf-container"
         url="pdfUrl"
         scale="1"
-        show-toolbar="true"></pdf-viewer>
+        show-toolbar="true"
+        headers="{'x-you-know-whats-awesome': 'EVERYTHING'}"></pdf-viewer>
```

Comment:
![awesome](https://bostongazette.files.wordpress.com/2014/02/the-lego-movie-awesome-e1392309318427.png?w=600)